### PR TITLE
[readme] Update Callback section with python code snippets

### PR DESCRIPTION
**Pull request by LastMileNewGrad**
[readme] Update Callback section with python code snippets

title

### CODE REVIEW

**Comment by LastMileNewGrad** [comment]
https://github.com/lastmile-ai/aiconfig/assets/141073967/ce909fc4-881f-40d9-9c67-78a6682b3063

File: README.md [hunk 1]
```diff
@@ -352,6 +357,10 @@ const myLoggingCallback: Callback = async (event: CallbackEvent) => {
   console.log(`Event triggered: ${event.name}`, event);
 };
 ```
+```python
+async def my_logging_callback(event: CallbackEvent) -> None:
+  print(f"Event triggered: {event.name}", event)
```
Comment (saqadri) [inline README.md:362]: Show sample output

File: README.md [hunk 2]
```diff
@@ -343,6 +343,11 @@ const customCallbackManager = new CallbackManager([yourCustomCallback]);
 aiConfigRuntimeInstance.setCallbackManager(customCallbackManager);
 ```
 
+```python
+custom_callback_manager = CallbackManager([yourCustomCallback])
```
Comment (saqadri) [inline README.md:347]: What is yourCustomCallback? This code sample is incomplete. It shouldn't be pseudo code, fully runnable example

Comment (saqadri) [inline README.md:347]: What is yourCustomCallback? This code sample is incomplete. It shouldn't be pseudo code, fully runnable example

File: README.md [hunk 3]
```diff
@@ -399,7 +427,10 @@ The CallbackManager uses a timeout mechanism to ensure callbacks do not hang ind
 const customTimeout = 10; // 10 seconds
 const callbackManager = new CallbackManager(callbacks, customTimeout);
 ```
-
+```python
+custom_timeout = 10; # 10 seconds
+callback_manager = CallbackManager([my_logging_callback], custom_timeout)
+```
```
Comment (saqadri) [inline README.md:433]: Can you add link to your video in readme? I'll move it after

**Review by saqadri — APPROVED** [review]
Approving to unblock, some comments

File: README.md (hunk 1 above)
Comment (LastMileNewGrad) [reply README.md:362]: updated with sample output

File: README.md (hunk 2 above)
Comment (LastMileNewGrad) [reply README.md:347]: updated to define callback and use it

Comment (LastMileNewGrad) [reply README.md:347]: updated to define callback and use it

File: README.md (hunk 3 above)
Comment (LastMileNewGrad) [reply README.md:442]: Added raw link in beggining of section, ctrl-f "video: "

**Comment by saqadri** [comment]
@LastMileNewGrad going to merge this in so I can keep working off it!

**Pull request merged**